pkg/soc/soc.go

```diff
@@ -22,130 +21,118 @@ const (
 	minChunkSize  = IdSize + SignatureSize + swarm.SpanSize
 )
 
-// Id is a soc identifier
-type Id []byte
-
-// Owner is a wrapper that enforces valid length address of soc owner.
-type Owner struct {
-	address []byte
-}
+var (
+	errInvalidAddress = errors.New("soc: invalid address")
+	errWrongChunkSize = errors.New("soc: chunk length is less than minimum")
+)
 
-// NewOwner creates a new Owner.
-func NewOwner(address []byte) (*Owner, error) {
-	if len(address) != crypto.AddressSize {
-		return nil, fmt.Errorf("invalid address %x", address)
-	}
-	return &Owner{
-		address: address,
-	}, nil
-}
+// ID is a soc identifier
+type ID []byte
 
-// Soc wraps a single soc.
+// Soc wraps a single-owner chunk.
```

Comment:
actually, `Soc` wraps a content-addressed chunk :)

Reply:
haha of course, sorry. Fixed.